### PR TITLE
Feature/bi 7582 validaiton why object screen

### PR DESCRIPTION
**Pull request by mouhajer-ch**
Add validation logic to enter information page
Generalise validation logic to be more DRY

Resolve [BI-7582](https://companieshouse.atlassian.net/browse/BI-7582)

### CODE REVIEW

File: views/enter-information.html [hunk 1]
```diff
@@ -17,25 +17,18 @@
...
 
-        {% if errInformation.flag %}
-          {% set informationError = {
-            text: errInformation.text
-          } %}
-        {% else %}
-          {% set informationError = false %}
+        {% if errorList.length > 0 %}
+          {{ govukErrorSummary({
+            titleText: ERROR_TEXT_BOX_EMPTY,
```
Comment (toriwilliamsch) [inline views/enter-information.html:29]: This needs to be `ERROR_SUMMARY_TITLE`

File: src/app.ts [hunk 2]
```diff
@@ -30,6 +30,7 @@ env.addGlobal("assetPath", process.env.CDN_HOST);
 env.addGlobal("PIWIK_URL", process.env.PIWIK_URL);
 env.addGlobal("PIWIK_SITE_ID", process.env.PIWIK_SITE_ID);
 env.addGlobal("ERROR_SUMMARY_TITLE", ErrorMessages.ERROR_SUMMARY_TITLE);
+env.addGlobal("ERROR_TEXT_BOX_EMPTY", ErrorMessages.ERROR_TEXT_BOX_EMPTY);
```
Comment (toriwilliamsch) [inline src/app.ts:33]: We can remove this global variable as this is specific to one page only (`enter-information`)

File: src/model/error.messages.ts [hunk 3]
```diff
@@ -14,6 +14,7 @@ export enum ErrorMessages {
   ERROR_404 =  "Page not found",
   ERROR_500 = "An internal server error has occurred",
   ERROR_SUMMARY_TITLE = "There is a problem",
+  ERROR_TEXT_BOX_EMPTY = "Enter a reason for your objection",
```
Comment (toriwilliamsch) [inline src/model/error.messages.ts:17]: Can this be renamed to something like `ERROR_REASON_EMPTY` please?

File: views/enter-information.html (hunk 1 above)
Comment (mouhajer-ch) [reply views/enter-information.html]: Thanks Tori! Fix done.

File: src/app.ts (hunk 2 above)
Comment (mouhajer-ch) [reply src/app.ts]: Thanks Tori! Fix done.

File: src/model/error.messages.ts (hunk 3 above)
Comment (mouhajer-ch) [reply src/model/error.messages.ts]: Thanks Tori! Fix done.

**Review by toriwilliamsch — APPROVED** [review]

**Pull request merged**